src/views/other-reason.njk

```diff
@@ -51,6 +51,9 @@
             label: {
               text: 'Detailed description of the reason'
             },
+            hint: {
+              text: "You should include your name and relationship to the company"
```

Comment:
Hints in the penalty details page always have periods at the end, prob worth checking with Bart?

Reply:
I'l check with him

Reply:
Period added